org.mwc.cmap.legacy/src/MWC/GUI/JFreeChart/StepperXYPlot.java

```diff
@@ -219,6 +226,16 @@ public HiResDate getTimeZero() {
 	public boolean isGrowWithTime() {
 		return _growWithTime;
 	}
+	
+	private boolean _centreWindow;

```

Comment:
@saulhidalgoaular - could member variables go near the top of the class, please - and could you also put in a quick comment to explain what `centreWindow` behaviour is?